test/integration/FileAppendIntegrationTest.js

```diff
@@ -211,7 +215,67 @@ describe("FileAppend", function () {
         ).getReceipt(env.client);
     });
 
+    it("should keep content after deserialization", async function () {
+        this.timeout(120000);
```

Comment:
I really don't like that we have to do this every single time. Consider working around this or making it configurable.

Reply:
Maybe create an issue for it?

Reply:
We can work around this by editing the Taskfile that runs the integration tests.